sdk/src/test/java/com/google/cloud/dataflow/sdk/io/BigQueryIOTest.java

```diff
@@ -20,6 +20,8 @@
 import static org.junit.Assert.assertNull;
 
 import com.google.api.client.util.Data;
+import com.google.api.services.bigquery.model.TableFieldSchema;
+import com.google.api.services.bigquery.model.JobConfigurationLoad;
```

Comment:
Looks like there was an error in backport that reordered imports. Please fix -- we'd like to run a sync today and this is blocker.